_pages/crossbrowser-testing.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Cross-browser Testing
+---
+
+Regular cross-browser testing helps ensures your web site or application is accessible and functional to a broad audience, especially as Internet Explorer 11 is pervasive in federal government but TTS developers use Chrome on macOS almost exclusively.
+
+Using a responsive web design and a standards-based approach will result in wide compatibility across modern browsers, but at the edges your product can still trip up on browser-specific quirks. For example, compare the [list of web technologies supported in Internet Explorer 11 versus Chrome](https://caniuse.com/#compare=ie+11,chrome+86). [Progressive Web Application](https://web.dev/progressive-web-apps/)-related features and behaviors  also vary widely across devices and browsers.
+
+A few options are available at TTS that allow testing on multiple browser platforms. An automated testing solution is ideal, but even manual cross-browser testing remains valuable.
+
+## Windows
+
+To test with Windows-native browsers like Internet Explorer or Microsoft Edge: 
+
+- [Citrix Workspace](https://handbook.tts.gsa.gov/vmware-horizon/)
+  * Pros:
+     * No extra installation necessary
+  * Cons:
+     * You can’t pick the browser versions; the Standard Desktop image includes Internet Explorer 11 and Microsoft Edge (EdgeHTML layout engine) 
+     * You can’t access localhost
+     * Manual testing only
+- Virtual machine (VM)
+  * Request access to [VirtualBox](https://www.virtualbox.org) through the GSA Self Service app, download and install a Windows VM, then set up access localhost (if needed).
```

Comment:
As someone interested in this workflow, I cannot find the option to install VirtualBox through the GSA Self Service app. Am I missing something? Are there certain prerequisites (and if so, should they be noted here as well)?

Reply:
If it doesn't show up in Self Service, I think the next stop is the IT help desk. You might have to be added to a dev group or something that GSA IT manages. I'll add that extra bit of advice.